docs/dev/Development Workflow.md

```diff
@@ -28,8 +28,10 @@ yarn
 yarn start
 ```
 Once the project is served, you can explore the components 
```

Comment:
Could add that these are the ports by default, but if there are already in use, the packages will be served on the next available ones, that can be seen in the console
<img width="613" alt="Screenshot 2021-04-22 at 10 04 11" src="https://user-images.githubusercontent.com/15702139/115670446-1763e300-a352-11eb-8486-47ff301e4e5d.png">

Reply:
Done